### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by kuldipsinghrai**
#minor#
@rox163 
@vesper2000 
@dafortin 
@JasonYe007 
@d-ashfield

### CODE REVIEW

**Comment by rox163** [comment]
:+1:

**Comment by kuldipsinghrai** [comment]
Ok i need to update eslint to 2.9.0

**Comment by kuldipsinghrai** [comment]
I am confused now with this conflict now.

**Comment by rox163** [comment]
You should fetch upstream before any PR because Travis will always be pushing new commits to master after your merges, in order to update the version number in package.json.

File: README.md [hunk 1]
```diff
@@ -20,10 +20,17 @@
...
+```
+{{#frost-sidebar class="demo-instance"}}
+  <div class="demo-pod-content">
+    <h4>Test content</h4>
+    {{frost-checkbox  checked=true label="Check me"}}
+  </div>
+{{/frost-sidebar}}
+```
 
 ## Development
```
Comment (vesper2000) [inline README.md]: This should also include content, similar as other readmes

Comment (kuldipsinghrai) [inline README.md]: Can you please add a link of the example readme here please? I want to make sure I am looking at the proper example.

Comment (vesper2000) [inline README.md]: Most of the repos have it.  frost-core is a bit different as there is a main page and then a link to each individual component.  Take a look at this one: https://github.com/ciena-frost/ember-frost-modal-input

Comment (kuldipsinghrai) [inline README.md]: Readme does look like the one in the above link especially the development section. Am I totally confused?

Comment (vesper2000) [inline README.md]: Actually, I think it is good - I thought the text was not there as the PR did not seem to have any content.  I checked the full file and it looks good.  Sorry about it.

Comment (kuldipsinghrai) [inline README.md]: Not a problem at all. Thanks for the confirmation and prompt response.

**Pull request merged**